src/test/java/executor/service/model/WebDriverConfigTest.java

```diff
@@ -0,0 +1,63 @@
+package executor.service.model;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class WebDriverConfigTest {
+
+    @Test
+    void testDefaultConstructor() {
```

Comment:
let's use a common code style

```suggestion
   public void testDefaultConstructor() {
```